modules/nw-metallb-addresspool-cr.adoc

```diff
@@ -49,3 +49,31 @@ Specify `false` if you want explicitly request an IP address from this pool with
 The default value is `true`.
 
 |===
+
+You can assign IP addresses from an `IPAddressPool` to services and namespaces by configuring the `spec.serviceAllocation` specification.
+
+.MetalLB IPAddressPool custom resource spec.serviceAllocation subfields
```

Comment:
spec.allocateTo as it has sub fields like priority, namespaces, namespaceSelector and serviceSelector.

Reply:
nevermind it is not spec.allocateTo like in design example. It is spec.serviceAllocation.